tests/hooks_test.py

```diff
@@ -185,6 +188,8 @@ def test_python_rst_inline_touching_normal_positive(s):
         '``inline`` normal',
         'normal ``inline`` normal',
         'normal ``inline``',
+        '    ``indented``inline touching',
+        '> quoted ``inline``touching',
```

Comment:
these still look incorrect and I think should be flagged

Reply:
We didn't actually see any issues with `rst-inline-touching-normal` in https://github.com/python/peps/pull/1668/files, shall I just ditch the changes to this hook?

Reply:
yes